app/src/main/java/com/sebastianvm/mango/database/dao/UserDao.kt

```diff
@@ -0,0 +1,25 @@
+package com.sebastianvm.mango.database.dao
+
+import androidx.room.Dao
+import androidx.room.Delete
+import androidx.room.Insert
+import androidx.room.Query
+import com.sebastianvm.mango.database.models.UserEnt
+
+@Dao
+interface UserDao {
+    @Query("SELECT * FROM UserEnt")
+    suspend fun getAll(): List<UserEnt>
+
+    @Query("SELECT * FROM UserEnt WHERE id = :id LIMIT 1")
+    suspend fun getUserById(id: Int): UserEnt?
```

Comment:
My thoughts on this: we should not expose `suspend` read functions. We should always expose a flow. If we want to do a one-time read, we can always use `first()`, but there is no way to go from `suspend` to flow

Reply:
Oh interesting 🤔 is this supposed to be more idiomatic? versus having a suspend fun _and_ fun that returns a flow?

Reply:
I think for the most part, we're going to be reading the data in the `ViewModel` as something that we'll map to UI State, so we want to be reactive to that (i.e. we need a flow). Single shot reads will be less frequent, and I think that instead of just creating both the flow and the suspend methods, we can reuse the flow method with `first`. 

Not sure if this is the idiomatic way, or what the consensus or best practices are, but having the single flow function makes our interfaces simpler (one less thing to implement in fakes, too)